src/main/java/ru/ashebalkin/skypro/course2/lesson6/collections/service/CollectionServiceImpl.java

```diff
@@ -0,0 +1,88 @@
+package ru.ashebalkin.skypro.course2.lesson6.collections.service;
+
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+@Service
+public class CollectionServiceImpl implements CollectionService {
+
+    private final List<Integer> nums = new ArrayList<>(List.of(12, 112, 12, 22, 55, 88, 13, 1, 1, 2, 3, 4, 4, 5, 5, 6, 7));
+    private final List<String> words = new ArrayList<>(List.of("каждый", "охотник", "хочет", "дичь", "дичь", "хочет", "жить", "свободно", "охотник", "хочет", "есть"));
+
+    @Override
+    public List<Integer> getOddNumbers() {
+        List<Integer> oddNums = new ArrayList<>();
+
+        for (Integer num : nums) {
+            if (num % 2 != 0) {
+                oddNums.add(num);
+            }
+        }
+
+        return oddNums;
+    }
+
+    @Override
+    public List<Integer> removeOddNumbers() {
+        List<Integer> oddNums = new ArrayList<>(List.copyOf(nums));
+
+        oddNums.removeIf(integer -> integer % 2 == 0);
```

Comment:
именовать переменные их типом не лучший вариант, проще
```suggestion
        oddNums.removeIf(num -> num % 2 == 0);
```